index.html

```diff
@@ -9,7 +9,7 @@
     <meta name="description" content="Avocado - Automated Testing Framework">
     <meta name="author" content="">
 
-    <title>Avocado - Testing made easier</title>
+    <title>Avocado Testing Framework</title>
```

Comment:
Actually this only changes the window name. How about changing the main heading on line 71? Also you should probably adjust the meta description (line 9) in order to keep it in sync.